PARAMETERS_MAPPING/parameters.csv

```diff
@@ -1,170 +1,172 @@
+131;surface_net_upward_shortwave_flux;;;Net shortwave heat flux;;
+132;upward_heat_flux_in_air;;;Net heat flux;;
+133;upward_mass_flux_of_air;;;Net mass flux;;
+134;sea_surface_wave_mean_period;;;;;
+135;sea_surface_wave_significant_period;;;;;
+136;;;sea_surface_wave_root_mean_square_amplitude_from_variance_spectral_density;;;
+137;sea_surface_wave_period_at_variance_spectral_density_maximum;;;;;
+138;;;sea_surface_wave_period_at_second_largest_peak_of_variance_spectral_density;;;
+139;;;sea_surface_wave_variance_spectral_density_zeroth_frequency_moment;;;
+140;sea_surface_wave_from_direction;;;;;
+141;;;mole fraction of CO2 (in the equilibrator head space or in the atmosphere - see TYPE column);;;
+142;;;mole fraction of H2O;;;
+143;;;pressure in the infrared gas analyser;;;
+145;;;main laboratory seawater flow;;;
+146;;;carbon dioxide reference value;;;
+147;;;measurement type (equilibrator or standard or atmosphere);;;
+154;;;differential pressure between equilibrator and ambient air;;;
+155;sea_surface_wave_from_direction_at_variance_spectral_density_maximum;;;;;
+156;sea_surface_wave_directional_spread;;;;;
+157;;;Indicator for method of calculation of spectral data (WMO Code Table 1744: 1=Longuet-Higgins (1964), 2=Longuet-Higgins (F3 method), 3=Max likelihood method, 4=Max entropy method);;;
+158;;;Sampling interval frequency;;;
+159;;;Duration of record of wave;;;
+160;;;sea_surface_wave_spectral_significant_height;;;
+161;sea_surface_wave_mean_period_from_variance_spectral_density_first_frequency_moment;;;;;
+162;sea_surface_wave_period_of_highest_wave;;;;;
+163;sea_surface_wave_mean_period_from_variance_spectral_density_second_frequency_moment;;;;;
+164;;;root mean square wave height calculated from the spectra;;;
+165;;;spectral width from the time domain;;;
+166;;;spectral width calculated from the spectral moment;;;
+167;;;sea_surface_swell_wave_significant_height;;;
+168;;;sea_surface_swell_wave_period_at_variance_spectral_density_maximum;;;
+169;sea_surface_swell_mean_period_from_variance_spectral_density_first_frequency_moment;;;;;
+170;;;sea_surface_swell_wave_from_direction_at_variance_spectral_density_maximum;;;
+171;;;sea_surface_wind_wave_significant_height;;;
+172;;;sea_surface_wind_wave_period_at_variance_spectral_density_maximum;;;
+173;sea_surface_wind_wave_mean_period_from_variance_spectral_density_first_frequency_moment;;;;;
+174;;;sea_surface_wind_wave_from_direction_at_variance_spectral_density_maximum;;;
+175;;;mass_concentration_of_inferred_chlorophyll_from_relative_fluorescence_units_in_sea_water;;conversion from fluorescence to chlorophyll-a using unknown coefficients
+176;;;mass_concentration_of_inferred_chlorophyll_from_relative_fluorescence_units_in_sea_water;;conversion from fluorescence to chlorophyll-a using factory coefficients
+177;;;mass_concentration_of_inferred_chlorophyll_from_relative_fluorescence_units_in_sea_water;;conversion from fluorescence to chlorophyll-a using user coefficients
```

Comment:
The comments you added in the last three rows are actually in the `imos_vocabulary_id` column. You need one more semi-colon in there.